app/views/partner_users/invitations/edit.html.erb

```diff
@@ -0,0 +1,50 @@
+<div class='login-page login-page--partner'>
+  <div class="login-box">
+    <div class="login-logo">
+      <img id="MastheadLogo" src="/img/partnerbase-logo.svg" style="width: 80%; height: 80%; " alt="Partner" title="Partner" class="serv_icon"/>
+    </div>
+
+    <div class="login-box-body">
+      <div class="form-group has-feedback">
+        <%= render partial: "shared/flash" %>
+
+        <%= simple_form_for resource, as: resource_name, url: invitation_path(resource_name), html: { method: :put } do |f| %>
+
+          <div class="login-box">
+            <!-- /.login-logo -->
+            <div class="card">
+
+              <div class="card-body login-card-body">
+                <h2><%= t 'devise.invitations.edit.header' %></h2>
+                <p>Welcome to Partnerbase! Before we get started, please set a new password for yourself.</p>
+                <p>We recommend a password that is:</p>
+                <ul>
+                  <li>Memorable to you</li>
+                  <li>At least 10 characters long</li>
```

Comment:
hmm, we're not enforcing this right? maybe we should

Reply:
@albertchae that is correct. We can add an issue to start enforcing password strength. I can do that.